### PR TITLE
code sniffer fix

### PR DESCRIPTION
**Pull request by DmitriKadykov**

### CODE REVIEW

File: src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php [hunk 1]
```diff
@@ -10,9 +10,9 @@
 interface AkeneoResourcePageInterface
 {
     /**
-     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
+     * @return $this
      */
-    public function getFirstPage(): AkeneoResourcePageInterface;
+    public function getFirstPage(): self;
```
Comment (dereuromark) [inline src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php]: No typehints for self

File: src/SprykerEco/Shared/AkeneoPim/AkeneoPimConstants.php [hunk 2]
```diff
@@ -7,11 +7,14 @@
...
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
 interface AkeneoPimConstants
 {
-    public const HOST = 'AKENEOPIM:HOST';
-    public const USERNAME = 'AKENEOPIM:USERNAME';
-    public const PASSWORD = 'AKENEOPIM:PASSWORD';
-    public const CLIENT_ID = 'AKENEOPIM:CLIENT_ID';
-    public const CLIENT_SECRET = 'AKENEOPIM:CLIENT_SECRET';
+    public const HOST = 'AKENEO_PIM:HOST';
```
Comment (dereuromark) [inline src/SprykerEco/Shared/AkeneoPim/AkeneoPimConstants.php]: These changes possibly trigger a major. Are you sure this is in the scope for the current release?

Comment (DmitriKadykov) [reply src/SprykerEco/Shared/AkeneoPim/AkeneoPimConstants.php]: it wasn't critical for the problem i faced with. OK i'll roll it back

Comment (dereuromark) [reply src/SprykerEco/Shared/AkeneoPim/AkeneoPimConstants.php]: This kind of BC breaking technical depth please note down and we fix it in the next major.

File: src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php [hunk 3]
```diff
@@ -10,9 +10,9 @@
 interface AkeneoResourcePageInterface
 {
     /**
-     * @return $this
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
      */
-    public function getFirstPage(): self;
+    public function getFirstPage(): AkeneoResourcePageInterface;
```
Comment (dereuromark) [inline src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php]: no typehint on self.

File: src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php [hunk 4]
```diff
@@ -10,9 +10,9 @@
 interface AkeneoResourcePageInterface
 {
     /**
-     * @return $this
+     * @return \SprykerEco\Service\AkeneoPim\Dependencies\External\Api\Wrapper\AkeneoResourcePageInterface
```
Comment (dereuromark) [inline src/SprykerEco/Service/AkeneoPim/Dependencies/External/Api/Wrapper/AkeneoResourcePageInterface.php]: $this or self is correct here

**Pull request merged**